drivers/sensor/ina23x/ina237.c

```diff
@@ -25,13 +25,20 @@ LOG_MODULE_REGISTER(INA237, CONFIG_SENSOR_LOG_LEVEL);
 /** @brief Power scaling (scaled by 10) */
 #define INA237_POWER_SCALING 2
 
+/**
+ * @brief Scale die temperture from 0.125 degC/bit to micro-degrees C
+ *  Note that the bottom 4 bits are reserved and are always zero.
+ */
+#define INA237_DIETEMP_TO_uDegC(x) ((x >> 4) * 125000)
```

Comment:
There should be `()` around the `x` in the macro.  Otherwise consider what happens if someone writes  `INA237_DIETEMP_TO_uDegC(data->die_temp & ~0x000F)` or something like that.

Reply:
Good catch, thank you.

Reply:
Fixed, will be in next patchset.